Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -1795,6 +1796,66 @@ def delete_alert_rule_command(client: AzureSentinelClient, args: Dict[str, Any])
     return CommandResults(readable_output=f'Alert rule {rule_id} was deleted successfully.')
 
 
+def list_subscriptions_command(client: AzureSentinelClient) -> CommandResults:
+
+    full_url = 'https://management.azure.com/subscriptions?api-version=2020-01-01'
+
+    response = client.http_request('GET', full_url=full_url)
+    data_from_response = response.get('value', [])
+
+    readable_output = tableToMarkdown(
+        'Azure Sentinel Subscriptions',
+        data_from_response,
+        ['subscriptionId', 'tenantId', 'displayName', 'state'], removeNull=True,
+        headerTransform=string_to_table_header)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='AzureSentinel.Subscription',
+        outputs=data_from_response,
+        outputs_key_field='subscriptionId',
+        raw_response=response
+    )
+
+
+def list_resource_groups_command(client: AzureSentinelClient, args: Dict[str, Any], subscription_id: str) -> CommandResults:
+
+    tag = args.get('tag')
+    limit = arg_to_number(args.get('limit', 50))
+    subscription_id_list = argToList(subscription_id)
+
+    # extracting the tag name and value from the tag argument that is received from the user as a string
+    filter_by_tag = ''
+    if tag:
+        try:
+            tag = json.loads(tag)
+            tag_name = next(iter(tag))
+            tag_value = tag[tag_name]
+            filter_by_tag = f"tagName eq '{tag_name}' and tagValue eq '{tag_value}'"
+        except Exception as e:
+            raise ValueError(('Invalid tag format, please use the following format: `{"key_name":"value_name"}`', e))
+
+    data_from_response = []
+    for subscription_id in subscription_id_list:
+        full_url = f'https://management.azure.com/subscriptions/{subscription_id}/resourcegroups?$filter={filter_by_tag}&$top={limit}&api-version=2021-04-01'   # noqa: E501
```

Comment:
why do we use a different api version in each command?

Reply:
by design